garden/tutorials.md

```diff
@@ -14,13 +17,9 @@ These tutorials cover general concepts to help get you started with Gazebo.
 * [Model Insertion from Fuel](fuel_insert)
 * [Keyboard Shortcuts](hotkeys)
 
-## Basics tutorials
+## ROS integration
 
-* [Building Your Own Robot](building_robot)
-* [Moving the Robot](moving_robot)
-* [SDF Worlds](sdf_worlds)
-* [Sensors](sensors)
-* [Actors](actors)
+* [ROS 2 Integration](ros2_integration)
 
 
 More specific content correlating to each Gazebo library can be found under the *API & Tutorials* sections on the [Libraries page](/libs).
```

Comment:
While we're doing sections here, can we add a heading for this so that it's more visible? Something like
```suggestion
## Per-library tutorials
More specific content correlating to each Gazebo library can be found under the *API & Tutorials* sections on the [Libraries page](/libs).
```

Reply:
I just realized the other comment doesn't apply, so I'll take this to a new PR.